tools/backups2datalad.req.txt

```diff
@@ -7,4 +7,5 @@ datalad
 #fscacher
 humanize
 morecontext ~= 0.1
+packaging
```

Comment:
Why not to use semantic-version package we already use in dandi CLI?

Reply:
We also use `packaging` in dandi-cli, and as long as version IDs are of the form `N.N.N`, the two packages will give the same results.

Reply:
indeed we do, but IMHO for no good reason:
```
$> git grep packaging
dandi/cli/cmd_shell_completion.py:from packaging.version import Version
setup.cfg:    packaging

$> git grep semantic_version
dandi/pynwb_utils.py:import semantic_version
dandi/pynwb_utils.py:    if isinstance(v, str) and not semantic_version.validate(v):
dandi/tests/test_metadata.py:from semantic_version import Version
dandi/tests/test_utils.py:from semantic_version import Version
dandi/utils.py:from semantic_version import Version
```

do you remember why to start using `packaging.version.Version`? if there is no advantage/need, I would say we should just use `semantic_version` and also remove `packaging` as a requirement for dandi-cli

Reply:
@yarikoptic dandi uses `packaging.version.Version` in order to test the version of Click installed.  Python package versions follow a different format than semver, and so `packaging` is the correct tool for the job there.